seed/challenges/03-front-end-libraries/bootstrap.json

```diff
@@ -43,9 +43,9 @@
         "With responsive design, there is no need to design a mobile version of your website. It will look good on devices with screens of any width.",
         "You can add Bootstrap to any app by adding the following code to the top of your HTML:",
         "<code>&#60;link rel=\"stylesheet\" href=\"//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css\"/&#62;</code>",
-        "In this case, we've already added it for you to this page behind the scenes.",
+        "In this case, we've already added it for you to this page behind the scenes. Note that using either of <code>></code> or <code>/></code> to close the <code>link</code> tag is acceptable.",
         "To get started, we should nest all of our HTML in a <code>div</code> element with the class <code>container-fluid</code>.",
-        "<strong>Note</strong><br>Using either of both <code>></code> or <code>/></code> to close the <code>link</code> tag are acceptable."
+        "<strong>Note</strong><br>HTML elements is all the code in the editor after the closing <code>style</code> tag."
```

Comment:
The `style` element is also an HTML element. Instead of adding this as a note, I suggest changing the line above this. Something like "... all of our HTML (except our style element) in a ...`